iwyu_globals.cc

```diff
@@ -93,6 +93,9 @@ static void PrintHelp(const char* extra_msg) {
          "        file names (default: 80).\n"
          "   --no_comments: do not add 'why' comments.\n"
          "   --no_fwd_decls: do not use forward declarations.\n"
+         "   --tolerate_transitive: If a file uses symbols from both a.h and b.h\n"
+         "        and includes a.h, tolerate b.h both when it is included and \n"
+	 "        when\n it is not.\n"
```

Comment:
Misaligned, needs another leading space. And spurious \n after "when"

Reply:
Oops, nvim put a tab in there, and I didn't see that because it aligns with tab width 8.